sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -622,8 +653,9 @@ def expand(self, pcoll):
                 test_client=self.test_client,
                 temporary_tables=self.temp_tables,
                 additional_bq_parameters=self.additional_bq_parameters),
-            load_job_name_pcv, *self.schema_side_inputs).with_outputs(
-                TriggerLoadJobs.TEMP_TABLES, main='main')
+            load_job_name_pcv, self.is_streaming_pipeline,
```

Comment:
I wonder if we should pass `is_streaming_pipeline` at pipeline construction (i.e. in the constructor) rather than a side input. It would allow us to show it as .p.ex. display data.

Reply:
According to the guide to generating Job IDs for BigQuery[1], the Job ID should be a human-readable prefix with a GUID/UUID suffix(Eg: `daily_import_job_1447971251`).
If we can generate the job ID as:
```python
"{}_{}".format(load_job_name_prefix, str(uuid.uuid4()))
```
 we can do away with passing the is_streaming_pipeline altogether. 
What are your thoughts? Will it make it harder for users to debug their jobs?

[1] https://cloud.google.com/bigquery/docs/running-jobs#generate-jobid